201-timeseriesinsights-environment-with-longterm-storage/azuredeploy.json

```diff
@@ -0,0 +1,294 @@
+{

```

Comment:
same changes will apply to this sample as in the other... will only add new/addition comments from here on